src/pl/plpython/plpy_main.c

```diff
@@ -162,18 +194,23 @@ PLy_initialize(void)
 		ereport(FATAL,
 				(errmsg("multiple Python libraries are present in session"),
 				 errdetail("Only one Python major version can be used in one session.")));
-
+#if PY_MAJOR_VERSION >= 3
+	/* PYTHONPATH and PYTHONHOME has been set to GPDB's python2.7 in Postmaster when
+	 * gpstart. So for plpython3u, we need to unset PYTHONPATH and PYTHONHOME.
+	 * if user set PYTHONPATH then we set it in the env
+	 */
+	if (plpython3_path && *plpython3_path) 
+		setenv("PYTHONPATH", plpython3_path, 1);
+	else 
+		unsetenv("PYTHONPATH");
+#endif
+	unsetenv("PYTHONHOME");
```

Comment:
I am not quite sure if there is anyone relies on the `PYTHONHOME` for plpython2 for now. Probably it will be safer to have this for python3 only.

Reply:
copy that.